lib/web_ui/lib/src/engine/semantics/selectable.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:ui/ui.dart' as ui;
+
+import 'label_and_value.dart';
+import 'semantics.dart';
+
+/// Renders semantics objects that have selected (true/false) states.
+///
+/// See also [ui.SemanticsFlag.isSelected]
+class SemanticSelectable extends SemanticRole {
```

Comment:
Selectable has special meaning in framework. It means it can be selected like text using mouse gesture.

Maybe name this SemanticsTapSelectable or SemanticsClickSelectable.